modules/appeals_api/spec/factories/notice_of_disagreements.rb

```diff
@@ -107,15 +116,46 @@
   factory :notice_of_disagreement_v0,
           class: 'AppealsApi::NoticeOfDisagreement', parent: :notice_of_disagreement_v2 do
     api_version { 'V0' }
+    auth_headers do
+      JSON.parse(File.read(::Rails.root.join(
+        *'/modules/appeals_api/spec/fixtures/notice_of_disagreements/v0/valid_10182_headers.json'.split('/')
+      ).to_s))
+    end
+    form_data do
+      JSON.parse(File.read(::Rails.root.join(
+        *'/modules/appeals_api/spec/fixtures/supplemental_claims/v0/valid_10182.json'.split('/')
```

Comment:
I was just looking this over and realized I made a bad copy/paste here with the filename (I think tests are still passing just because the v0 factories are barely used yet) - will fix momentarily

Reply:
Ok, fixed these and double checked the other v0 factories' paths just to make sure